tests/integration/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarFunctionsTest.java

```diff
@@ -2167,4 +2173,61 @@ private  void testDebeziumPostgreSqlConnect() throws Exception {
         getSourceInfoNotFound(tenant, namespace, sourceName);
     }
 
+    private  void testDebeziumMongoDbConnect() throws Exception {
+
+        final String tenant = TopicName.PUBLIC_TENANT;
+        final String namespace = TopicName.DEFAULT_NAMESPACE;
+        final String outputTopicName = "debe-output-topic-name";
+        final String consumeTopicName = "public/default/dbserver1.inventory.products";
+        final String sourceName = "test-source-connector-"
+                + functionRuntimeType + "-name-" + randomName(8);
+
+        // This is the binlog count that contained in postgresql container.
+        final int numMessages = 26;
```

Comment:
In the scripts https://github.com/debezium/docker-images/blob/master/examples/mongodb/1.0/init-inventory.sh,
Number of data is 17, not is 26